i18n/es/docusaurus-plugin-content-docs/current/intro.md

```diff
@@ -0,0 +1,80 @@
+sobre machine learning. Pero si no tienes idea de lo que machine learning significa, lee la sección de Introducción básica.
+
+## El punto más importante de este curso es tu feedback!
+Si tienes cualquier consulta, comentario o sugerencia, por favor crea un **[issue](https://github.com/trigaten/Learn_Prompting/issues/new/choose)**, envíame un email a learnprompting@gmail.com o escribe un DM a [Discord](https://learnprompting.org/discord)/[Twitter](https://twitter.com/learn_prompting).
+
+Incluso el más mínimo feedback es de mucha ayuda!
+
+## Filosofía del curso
+
+**Iteraciones rápidas** - Dado que se publica nuevo contenido sobre PE casi a diario,
+Se estará actualizando este curso frecuentemente con articulos cortos sobre nuevas técnicas.
+Si quieres saber más sobre esto házmelo saber
+
+Parte de esta filosofía es **Iteración de errores**. Si en algún momento revisas algo
+que no entiendes, incluso algo pequeño, eso depende completamente de mi. **Por favor abre un [issue](https://github.com/trigaten/Learn_Prompting/issues/new/choose)**!
+
+
+**Foco en el sentido práctico** - Me centraré en técnicas prácticas y aplicadas que puedes utilizar
+inmediatamente para ser aplicadas.
+
+**Ejemplos de inmediato** - Intento poner ejemplos en los artículos siempre,
+para que pueda familiarizarse con las técnicas de una forma inmediata
+
+
+Filosofaré más sobre esto cuando tenga tiempo 😊
+
+## Cómo leer el curso
+
+No es necesario leer todos los capítulos en orden. ¡Lee lo que te interesa!
+
+Si eres totalmente nuevo, lee desde el inicio y luego comienza con la sección de Conceptos básicos.
+Si no, puede ser más útil comenzar con la sección intermedia.
+
+Los artículos están clasificados por dificultad y están etiquetados con lo siguiente:
+
+🟢 Very easy; no requiere programación
+
+🟡 Easy; Se requiere programación simple, pero sin mucha experiencia.
+
+🔴 Medium; se requiere programación y algo de experiencia (por ejemplo, problemas con registro de logs)
+
+🟣 Hard; Se requiere programación y una sólida experiencia en el dominio para implementar (por ejemplo, enfoques de aprendizaje por refuerzo)
+
+Nota: aunque para los problemas de 🔴🟣 la experiencia en el dominio es útil, por lo general aún podrás comprender el artículo.
+
+## Capítulos
+
+Te dejo este breve resumen de cada capítulo:
+
+**Básico**: Introducción a PE y técnicas simples de PE
+
+**Intermedio**: Técnicas de PE un poco más complicadas
+
+**Aplicación Avanzada**: Aplicar técnicas de PE muy potentes y más avanzadas
+
+**Prompting Aplicado**: Algunos tutoriales completos del proceso de PE escritos por miembros de la comunidad
+
+**Fiabilidad**: Cómo hacer que los LLMs sean más confiables
+
+**Imágenes**: ¡PE para modelos de texto a imagen como DALLE y Stable Diffusion!
+
+**Prompt Injection**: Hacking, pero para PE
+
+**Prompting IDEs**: Diferentes herramientas de PE
+
+**Prompt Tuning**: Fine tune prompts con degradados
```

Comment:
I took the liberty of reviewing the text with https://languagetool.org/ and made some minor adjustments.

---
sidebar_position: 1
---
# Bienvenidos

¡Bienvenidos a este curso de Ingeniería en Prompt!

Me gusta pensar que la Ingeniería en Prompt (PE) es: **Cómo hablar con una AI para obtener lo que quieres**. 


Con tantos avances en inteligencia artíficial (AI), esta se convierte en una habilidad particularmente importante. Este curso pone el foco en aplicar técnicas de PE. Se esperan unos mínimos conocimientos sobre machine learning. Pero si no tienes idea de lo que machine learning significa, lee la sección de Introducción básica.

## ¡El punto más importante de este curso es tu feedback!
Si tienes cualquier consulta, comentario o sugerencia, por favor crea un **[issue](https://github.com/trigaten/Learn_Prompting/issues/new/choose)**, envíame un email a learnprompting@gmail.com o escribe un DM a [Discord](https://learnprompting.org/discord)/[Twitter](https://twitter.com/learn_prompting).

¡Incluso el más mínimo feedback es de mucha ayuda!

## Filosofía del curso

**Iteraciones rápidas** - Dado que se publica nuevo contenido sobre PE casi a diario, actualizaré este curso frecuentemente con artículos cortos sobre nuevas técnicas. ¡Si quieres saber más sobre esto házmelo saber!

Parte de esta filosofía es **Iteración de errores**. Si en algún momento revisas algo que no entiendes, incluso algo pequeño, eso depende completamente de mí. **¡Por favor, abre un [issue](https://github.com/trigaten/Learn_Prompting/issues/new/choose)**!

**Foco en lo práctico** - Me centraré en técnicas prácticas y aplicadas que puedes usar inmediatamente en tus aplicaciones.

**Ejemplos de inmediato** - Intento siempre poner ejemplos en los artículos, para que puedas familiarizarte con las técnicas de una forma inmediata.

Filosofaré más sobre esto cuando tenga tiempo 😊

## Cómo leer el curso

No es necesario leer todos los capítulos en orden. ¡Lee lo que te interesa!

Si eres totalmente nuevo, lee desde el inicio y luego comienza con la sección de Conceptos básicos.
Si no, puede ser más útil comenzar con la sección intermedia.

Los artículos están clasificados por dificultad y están etiquetados con lo siguiente:

🟢 Very easy; no requiere programación

🟡 Easy; se requiere programación simple, pero no experiencia en el dominio.

🔴 Medium; se requiere programación y algo de experiencia en el dominio (por ejemplo, problemas con registro de logs)

🟣 Hard; se requiere programación y una sólida experiencia en el dominio para implementar (por ejemplo, enfoques de aprendizaje por refuerzo)

Nota: aunque para los problemas 🔴🟣 la experiencia en el dominio es útil, por lo general aún así podrás comprender el artículo.

## Capítulos

Te dejo este breve resumen de cada capítulo:

**Básico**: Introducción a PE y técnicas simples de PE

**Intermedio**: Técnicas de PE un poco más complicadas

**Aplicación Avanzada**: Aplicar técnicas de PE muy potentes y más avanzadas

**Prompting Aplicado**: Algunos tutoriales completos del proceso de PE escritos por miembros de la comunidad

**Fiabilidad**: Cómo hacer que los LLMs sean más confiables

**Imágenes**: ¡PE para modelos de texto a imagen como DALLE y Stable Diffusion!

**Prompt Injection**: Hacking, pero para PE

**Prompting IDEs**: Diferentes herramientas de PE

**Prompt Tuning**: Fine tune prompts con degradados